### ISSUE TITLE
Does this cache images to disk or in memory?

### ISSUE THREAD

**Issue by CDBridger** [DylanVann/react-native-fast-image#776]
i.e If I view or preload an image from a url, quit the app, go offline, open the app. Will I still see the image? If not is it possible to make that happen?

**Comment by darrylyoung** [comment]
Yeah, you'll still see the image if you've already loaded it and then you open up your app again, offline. I just started using this component simply because React Native's built-in Image component was unreliable when it came to caching. This seems to work out of the box.

**Comment by dandre-hound** [comment]
Technically this package caches in memory by default. There is no way to change this without changing the Glide module config and SDWebImage config. Storing in memory is faster, but can take away memory from other memory-intensive features of your application, especially if you're caching large images. Storing in disk is a bit slower (_may_ result in a brief flash while loading if you don't configure it properly), but leaves more memoyr for the rest of your app.

**Comment by CDBridger** [comment]
Is there any where I can find details on the SDWebImage config? I have an offline first app so need the images to be cached to disk.

**Comment by dandre-hound** [comment]
@CDBridger have you checked their [Common Problems](https://github.com/SDWebImage/SDWebImage/wiki/Common-Problems#optimization-for-large-images) wiki?

**Comment by KarandikarMihir** [comment]
Where can I find docs for enabling disk storage on android (using Glide of course)?

**Comment by nes123** [comment]
Can we limit the memory size this package can allocate for images?
